src/FlowScene.cpp

```diff
@@ -351,6 +351,24 @@ connections() const
   return _connections;
 }
 
+std::vector<Node*>
+FlowScene::selectedNodes() const {
+  QList<QGraphicsItem*> graphicsItems = selectedItems();
+
+  std::vector<Node*> ret;
+  ret.reserve(graphicsItems.size());
+
+  for (QGraphicsItem* item : graphicsItems) {
+    auto ngo = qgraphicsitem_cast<NodeGraphicsObject*>(item);
```

Comment:
Well, it is Qt's decision. I just wonder what would they do with dozens of different graphical items. It must be a mess in defining the `type()` function and enums correctly